docs: updates to Pie chart documentation (#91608)
grafana/grafana: docs/sources/panels-visualizations/visualizations/pie-chart/index.md

{{< docs/play title="Grafana Bar Charts and Pie Charts" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}

## Supported data formats

> The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.

Reviewer: ```suggestion
The pie chart visualization requires a dataset with at least one row of numeric values to be added together to make up the complete pie. If the first row contains column titles, they're used to label the slices of the pie chart and appear in the legend. Cells with non-numeric values are not displayed in the pie chart.
```

> The pie chart panel requires a data set with at least one row with numeric values to be added up. If the first row contains column titles it will be used to display and identify the values in the pie chart. Cells with non-numeric values will not be displayed inside of the pie chart.

Reviewer: We always call these visualizations vs panels. Panels include a visualization as well as data and any transformations you apply. We wanna be really specific about this going forward. 

```suggestion
The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.
```

Example:

| Description   | Internet Explorer | Chrome |
| ------------- | ----------------- | ------ |

> | Other Users   | 1.5               | 7      |

Reviewer: ```suggestion
| Other users   | 1.5               | 7      |
```

> | English users | 17.5              | 50     |

Reviewer: I wonder if it makes sense to start with a simpler example where all the content will show up in the pie chart and then add this as a second example. And explain what will and will not appear in the chart in each example. For example:

> | Internet Explorer | Chrome |
> ----------------------------
> | 17  | 50 |
> 
> In this example you'll get a pie chart that looks like this (add image). 

I also wonder if you can add information about use cases for a pie chart; what kind of data you'd want to display this way. The dataset in the example doesn't seem like the type of thing you'd plug into a pie chart because you're trying to show multiple dimensions, I think, but I'm not sure.

Also, how do you have a half a user?

> | English Users | 17.5              | 50     |

Reviewer: ```suggestion
| English users | 17.5              | 50     |
```

> If the dataset contains multiple rows, the visualization displays the last row by default, but multiple rows can be configured as described in the [Value options](#value-options) section. If you configure multiple rows to be displayed, the panel concatenates the text fields of each row with the numeric column titles.

Reviewer: Can you show explicitly how you get multiple rows to be shown in the pie chart? And maybe even show an example.

Author: I will add images for that. I will try to keep them as generic Grafana pie as possible to avoid image maintenance.

> If the data set contains multiple rows the panel will display the last row by default, but multiple rows can be configured as described in the [Value options](#value-options) section. If you configure multiple rows to be displayed the panel will concatenate the text fields of each row with the numeric column titles.

Reviewer: ```suggestion
If the dataset contains multiple rows, the visualization displays the last row by default, but multiple rows can be configured as described in the [Value options](#value-options) section. If you configure multiple rows to be displayed, the panel concatenates the text fields of each row with the numeric column titles.
```

> It is recommended to use only one data set with the pie chart, if multiple data sets are present the numeric values configured in the _Value options_ section will be added up and displayed from all the data sets.

Reviewer: I've edited this a bit, but I'm still not exactly sure what it means. Do you mean the numeric values of all the datasets configured in the Value options section will be added together?

```suggestion
We recommend you only use one data set with the pie chart. If you have multiple datasets, the numeric values configured in the **Value options** section will be added up and displayed from all the datasets.
```
